index.js

```diff
@@ -378,4 +378,17 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+    name: 'Khang Huu Nguyen',
+    title: 'Student @ UMass Amherst',
+    hobbies: 'J-pop, Gaming, Anime',
```

Comment:
can we use lowercase for `gaming` and `anime` to be consistent with others above